lisp/mastodon-toot.el

```diff
@@ -35,18 +35,46 @@
   :prefix "mastodon-toot-"
   :group 'mastodon)
 
-(defvar mastodon-toot--reply-to-id nil)
-(defvar mastodon-toot--content-warning nil)
+(defcustom mastodon-toot-character-limit 500
+  "Maximum number of characters allowed in a toot."
+  :type 'number)
+
+(defun mastodon-toot--edit-info (&optional docs user)
+  "Take an optional DOCS string and USER handle and create an alist representing data about the edit buffer."
```

Comment:
Could you document what these values in the alist are meant to be?

It is really hard if you have to find their uses and from there try to divine what these value represent.